core/plugins/org.polarsys.capella.core.projection.exchanges/src/org/polarsys/capella/core/projection/exchanges/NodePhysicalComponentExchangesCreator.java

```diff
@@ -302,8 +305,13 @@ protected boolean doesNodeAlreadyHaveAPhysicalLinkForComponentExchange(PhysicalC
     for (Setting setting : inverseReferences) {
       // Search for a relation targeting the ComponentExchangeAllocation metaclass.
       if (setting.getEObject() instanceof ComponentExchangeAllocation) {
-        result = true;
-        break;

```

Comment:
Only for actors seems to be allowed to apply PL allocator at same level with CE, are this lines of code needed here since PhysicalLinksCreator needs to handle this case?